esmvalcore/preprocessor/_io.py

```diff
@@ -364,6 +394,11 @@ def _concatenate_overlapping_cubes(cubes):
     # get time end points
     time_1 = cubes[0].coord('time')
     time_2 = cubes[1].coord('time')
+    if time_1.units != time_2.units:
+        raise ValueError(
+            f"Cubes {cubes[0]} and {cubes[1]} can not be concatenated: "
```

Comment:
```suggestion
            f"Cubes\n{cubes[0]}\nand\n{cubes[1]}\ncan not be concatenated: "
```
will be easier to read